docs/stories/debugging/vscode/launch.json

```diff
@@ -2,7 +2,7 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
+    "version": "0.2.1",
```

Comment:
I think this version is a VSCode setting, not a "launch.json" semver version for our use. Can you revert it?

Reply:
reverted